script/check-collections.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+
+const defaultCollections = require('../script/collections/default.json');
+const brandConsolidationCollections = require('../script/collections/brand-consolidation.json');
+
+const checkCollections = buildOptions => (files, metalsmith, done) => {
+  const collection = buildOptions['brand-consolidation-enabled']
+    ? Object.keys(brandConsolidationCollections)
+    : Object.keys(defaultCollections);
```

Comment:
Is there a clearer name we could use for `collection`? It seems like it could be clearer to initialize it above the `(files, metalsmith, done) => ` function as `VALID_COLLECTIONS`, unless I'm reading it wrong.

Reply:
Yes I can do that.